src/Bicep.Core/Emit/ParamsFileTemplateWriter.cs

```diff
@@ -11,14 +11,16 @@
 
 namespace Bicep.Core.Emit
 {
-    public class ParamsFileWriter
+    public class ParamsFileTemplateWriter
```

Comment:
[nit] We have a 'template file' and a 'parameters file' - they are different things. This class is just for writing the parameters file - it feels more accurate to call this the `ParamsFileWriter` than to call it the `ParamsFileTemplateWriter`.

Reply:
Yes, it was initially ```ParamsFileWriter```. However as I started working on the cli, I had conflicts between [cli service](https://github.com/Azure/bicep/blob/Usman0111/compile-params-to-json-cli-logic/src/Bicep.Cli/Services/ParametersFileWriter.cs) and [emitter](https://github.com/Azure/bicep/blob/Usman0111/compile-params-to-json-cli-logic/src/Bicep.Core/Emit/ParamsFileTemplateWriter.cs) so I decided to follow the author of parameter action feature did (call the emitter file template writer instead of just writer)